<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="ReservaDeSala">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>541</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>EQUIPAMENTOS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CURSO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PERIODO:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFazerReserva">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RESERVA</string>
     </property>
    </widget>
    <widget class="QLabel" name="nomeDocenteResponsavel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NOME DOCENTES/RESPONSAVEL:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SALA:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RESERVA  DE SALA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="observacaoReserva">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>431</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OBSERVAÇÕES:</string>
     </property>
    </widget>
    <widget class="QLabel" name="feedbackReserva">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>440</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="nomeDocente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="salaReserva">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="equipamentosReserva">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cursoReserva">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="inicioReserva">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>INICIO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FIM:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="fimReserva">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
